src/preact10-internals.ts

```diff
@@ -1,5 +1,8 @@
+import { Fragment } from 'preact';
 import type { Component, VNode } from 'preact';
 
+Fragment.displayName = 'Fragment';
```

Comment:
This makes enzyme debug output easier to work with by giving Fragments a name that isn't minified